liferay-content-management/liferay-portal
# LPD-28771 Remove Feature Flag for the Epic

### What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-28771

Delete release feature Flag LPD-25680
 
### How am I fixing it?

Deleting feature flag

### How can you verify that it works?

Disabling or deleting all feature flags that you have in your bundle and going to content dashboard and also check if the requirement of the epic are working as expected:https://liferay.atlassian.net/browse/LPD-25680
